torch/_torch_docs.py

```diff
@@ -4390,7 +4390,9 @@ def merge_dicts(*dicts):
 
 Keyword arguments:
     {out}
-    {dtype}
+    dtype (torch.dtype, optional) : the desired data type of returned tensor. Default: if None,
+    and start and end are real values, uses a global default (see torch.get_default_tensor_type())
```

Comment:
This should reference set_default_dtype:

https://pytorch.org/docs/stable/generated/torch.set_default_dtype.html#torch-set-default-dtype

However I would just ignore that expert option and document this as follows:

> dtype (torch.dtype, optional) : the desired data type of returned tensor.

dtype (torch.dtype, optional): the data type to perform the computation in. Defaults to float when both :attr:\`start\` and :attr:\`end\` are real, and complex float when either is complex.

Reply:
From offline discussion with @anjali411: referencing get_default_dtype() would be better than set_default_dtype(), and it's OK to reference get_default_dtype() like torch.get_default_tensor_type() is referenced above.